build/prepare/CheckPrereqs.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="UbuntuCoreclrAndCoreFxDependencies.props" />
+
+	<Target Name="CheckPrereqs"
+			DependsOnTargets="CheckUbuntuAptGetDependencies;
+												CheckCentOsYumDependencies;">
+
+		<Message Text="Using Stage0 version:" />
+		<Exec Command="$(DotnetStage0) --version" />
```

Comment:
Just bringing along what we did before. This call will tell us that a Stage0 directory exists, that it has a dotnet, and that that dotnet has a version.